README.md

```diff
@@ -1,12 +1,8 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/9a24d489-6f78-40fa-91cd-4144d2580d0b/deploy-status)](https://app.netlify.com/sites/p2phackclub/deploys)
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=thatrobotdev/p2phackclub)](https://dependabot.com)
+
 <p align="center">
-  <a href="https://www.hackclub.com">
-    <img alt="P2P Hack Club Logo" src="static/logo.svg" width="60" />
-  </a>
```

Comment:
I think that we should keep this part in, and update the logo link because of it being broken. I think we can replace the link to the logo with one that is already on the website, but I can send you files of the logo if you need them 😄